librz/core/cil.c

```diff
@@ -609,58 +608,40 @@ RZ_IPI void rz_core_analysis_rzil_vm_status(RzCore *core, const char *var_name,
  * \return false if an error occured (e.g. invalid op)
  */
 RZ_IPI bool rz_core_rzil_step(RzCore *core) {
```

Comment:
And all those `rz_core_analysis_rzil_*` functions, to be consistent with naming changes you did.